alerting/src/test/kotlin/org/opensearch/alerting/resthandler/SecureMonitorRestApiIT.kt

```diff
@@ -476,4 +479,46 @@ class SecureMonitorRestApiIT : AlertingRestTestCase() {
             deleteRoleMapping("alerting_full_access")
         }
     }
+
+    fun `test admin all access with enable filter by`() {
```

Comment:
Since our GitHub Actions CICD is currently not running the security tests, could you capture in the PR somewhere that these tests were ran locally and that they passed?

Reply:
Tested and added